bundles/ch.elexis.core/src/ch/elexis/core/model/issue/ProcessStatus.java

```diff
@@ -4,6 +4,7 @@
 
 import ch.elexis.core.interfaces.ILocalizedEnum;
 import ch.elexis.core.interfaces.INumericEnum;
+import ch.elexis.core.model.MaritalStatus;
```

Comment:
Warum??

Reply:
Wenn Du diese Zeilen rausnimmst, kompiliert zwar alles, aber z.B. beim Anschauen der Status einer Erinnerung kommen, dann nur die nicht lokalisierten Konstanten. Grund dafür dürfte das folgende Diff sein
<pre>
git diff origin/master  bundles/ch.elexis.core/src/ch/elexis/core/model/issue/ProcessStatus.java
diff --git a/bundles/ch.elexis.core/src/ch/elexis/core/model/issue/ProcessStatus.java b/bundles/ch.elexis.core/src/ch/elexis/core/model/issue/ProcessStatus.java
<..>
@@ -31,8 +32,8 @@ public enum ProcessStatus implements INumericEnum, ILocalizedEnum {
 
        public String getLocaleText() {
                try {
-                       return ResourceBundle.getBundle("ch.elexis.core.model.issue.messages")
-                                       .getString(ProcessStatus.class.getSimpleName() + "." + this.name());
+                       return ResourceBundle.getBundle(ch.elexis.core.l10n.Messages.BUNDLE_NAME)
+                                       .getString(ProcessStatus.class.getSimpleName() + "_" + this.name());
                } catch (Exception e) {
                        return this.name();
                }
</pre>